pkgs/development/python-modules/streamz/default.nix

```diff
@@ -38,6 +39,9 @@ buildPythonPackage rec {
     requests
   ];
 
+  # zict dependency is now python 3.6+ only
```

Comment:
then dict should be marked accordingly, I wouldn't mark this as disabled just because a dependency doesn't work on it.

One example would be if someone override zict, streamz should still work.

Reply:
Thanks for the review! To clarify: You are saying that marking zict as `pythonOlder "3.6";` is enough?

Reply:
Ok thanks - updated accordingly.